admin_manual/installation/nginx-root.conf.sample

```diff
@@ -56,6 +56,12 @@ server {
     client_body_timeout 300s;
     fastcgi_buffers 64 4K;
 
+    # Other timeouts
+    proxy_connect_timeout 60s;
+    proxy_send_timeout 60s;
+    proxy_read_timeout 60s;
+    send_timeout 60s;
```

Comment:
```suggestion
    # Proxy and client response timeouts
    # Uncomment an increase these if facing timeout errors during large file uploads
    #proxy_connect_timeout 60s;
    #proxy_send_timeout 60s;
    #proxy_read_timeout 60s;
    #send_timeout 60s;
```
In case for the sub dir config respectively.